### PR TITLE
docs: refactor documentation to be inline with the new version

### PR DESCRIPTION
**Pull request by wolfenrain**
## Status

**READY**

## Description

Refactor documentation

## Type of Change

- [ ] ✨ New feature (non-breaking change which adds functionality)
- [ ] 🛠️ Bug fix (non-breaking change which fixes an issue)
- [ ] ❌ Breaking change (fix or feature that would cause existing functionality to change)
- [ ] 🧹 Code refactor
- [ ] ✅ Build configuration change
- [x] 📝 Documentation
- [ ] 🗑️ Chore

### CODE REVIEW

File: docs/docs/actions/custom.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+You can also pass extra options to the command to control the output, for the full overview of
+options run:
+
+```shell
+fluttium new action --help
+```
+
+## Finding published actions
+
+You can find actions for Fluttium that have been published by using [the dependency search from pub.dev](https://pub.dev/packages?q=dependency%3Afluttium)
```
Comment (felangel) [inline docs/docs/actions/custom.md:71]: 🔍

File: docs/docs/getting-started/initializing-fluttium.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+# the version of Fluttium that the project requires.
+environment:
+  fluttium: '>=0.1.0-dev.1 <0.1.0'
+
+# The driver can be configured with default values. Uncomment the following
+# lines to automatically run Fluttium using a different flavor and dart-defines.
+# driver:
+#   flavor: development
+#   dartDefines:
+#     - CUSTOM_DART_DEFINE=1
```
Comment (felangel) [inline docs/docs/getting-started/initializing-fluttium.md:28]: Do we still need this section?

Comment (wolfenrain) [reply docs/docs/getting-started/initializing-fluttium.md:28]: It is the generated output, to showcase that you can customize the driver through this file.

**Review by felangel — DISMISSED** [review]

**Review by novas1r1 — DISMISSED** [review]

File: docs/docs/actions/built-in/expectations.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+Fluttium provides a set of actions that will allow you to validate if certain aspects of your
+application are visible.
+
+## Visibility expectations
+
+To check if a specific text, through a semantic value or not, is visible, you can use the
+`expectVisible` action. This action goes through the semantic tree and checks semantic labels,
+tooltips, text nodes and other values to determine if something with the given text is visible.
+
```
Comment (scarletteliza) [inline docs/docs/actions/built-in/expectations.md:16]:
```suggestion
To check if a specific text is visible (through a semantic value or not), you can use the
`expectVisible` action. This action goes through the semantic tree and checks semantic labels,
tooltips, text nodes, and other values to determine if something with the given text is visible.

```

File: docs/docs/actions/built-in/interactions.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+The short-hand syntax for this action will assume you are using the text search:
+
+```yaml
+- pressOn: 'Your Text'
+```
+
+## Long press
+
+To emit a long press to the application you can use the `longPressOn` action. The full YAML syntax
+of this action is as followed:
```
Comment (scarletteliza) [inline docs/docs/actions/built-in/interactions.md:31]:
```suggestion
To emit a long press to the application you can use the `longPressOn` action. The full YAML syntax
of this action is as follows:
```

File: docs/docs/actions/custom.md [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+sidebar_position: 2
+---
+
+# 🎨 Custom Actions
+
+Fluttium supports custom actions, that you can install to be used in your user flows.
+
+Actions are defined as Dart packages, they can be published to [pub.dev](https://pub.dev) and have their own dependencies.
+
+You can add custom actions by adding them to the `actions` section in your `fluttium.yaml`:
```
Comment (scarletteliza) [inline docs/docs/actions/custom.md:11]:
```suggestion
Fluttium supports custom actions that you can install to be used in your user flows.

Actions are defined as Dart packages, which can be published to [pub.dev](https://pub.dev) and have their own dependencies.

You can add custom actions by adding them to the `actions` section in your `fluttium.yaml`:
```

File: docs/docs/actions/custom.md [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+
+## Creating a new action
+
+We can use the `new action` command to generate a custom action.
+
+```shell
+fluttium new action <action-name>
+```
+
+Once we run, `fluttium new create my_action`, we should see a `my_action` directory whose structure should look something like:
```
Comment (scarletteliza) [inline docs/docs/actions/custom.md:42]:
```suggestion
Once we run `fluttium new create my_action`, we should see a `my_action` directory whose structure should look something like:
```

File: docs/docs/getting-started/creating-a-flow-test.md [hunk 7]
```diff
@@ -0,0 +1,26 @@
...
+
+# 🪄 Creating A Flow Test
+
+We can use the `new flow` command to generate new flows for our project.
+
+```shell
+fluttium new flow <flow-name>
+```
+
+Once we run, `fluttium new flow my_flow`, we should see a `my_flow.yaml` that looks something like:
```
Comment (scarletteliza) [inline docs/docs/getting-started/creating-a-flow-test.md:13]:
```suggestion
Once we run `fluttium new flow my_flow`, we should see a `my_flow.yaml` that looks something like:
```

File: docs/docs/getting-started/creating-a-flow-test.md [hunk 8]
```diff
@@ -0,0 +1,26 @@
...
+
+```yaml
+description: My first Fluttium flow.
+---
+- log: 'Hello World!'
+```
+
+You can also pass extra options to the command to control the output, for the full overview of
+options run:
```
Comment (scarletteliza) [inline docs/docs/getting-started/creating-a-flow-test.md:22]:
```suggestion
You can also pass extra options to the command to control the output. For the full overview of
options run:
```

File: docs/docs/getting-started/initializing-fluttium.md [hunk 9]
```diff
@@ -0,0 +1,41 @@
+---
+sidebar_position: 2
+---
+
+# 📁 Initializing Fluttium
+
+Once `fluttium_cli` is installed, we can use the `init` command to initialize Fluttium in the
+current project. This command will generate a `fluttium.yaml` in the current directory. This allow
+you to install custom actions and configure the Fluttium driver.
+
```
Comment (scarletteliza) [inline docs/docs/getting-started/initializing-fluttium.md:10]:
```suggestion
Once `fluttium_cli` is installed, we can use the `init` command to initialize Fluttium in the
current project. This command will generate a `fluttium.yaml` in the current directory. This allows
you to install custom actions and configure the Fluttium driver.

```

File: docs/docs/getting-started/initializing-fluttium.md [hunk 10]
```diff
@@ -0,0 +1,41 @@
...
+```shell
+fluttium init
+```
+
+Once we run, `fluttium init`, we should have a `fluttium.yaml` that looks like:
```
Comment (scarletteliza) [inline docs/docs/getting-started/initializing-fluttium.md:15]:
```suggestion
Once we run `fluttium init`, we should have a `fluttium.yaml` that looks like:
```

File: docs/docs/getting-started/testing-a-first-flow-test.md [hunk 11]
```diff
@@ -0,0 +1,33 @@
+---
+sidebar_position: 4
+---
+
+# 🧪 Testing A Flow Test
+
+Testing a flow can be done through the `fluttium test` command. This command will read the
+`fluttium.yaml` file, apply any of the configuration it has to the driver and run the given flow
+file.
```
Comment (scarletteliza) [inline docs/docs/getting-started/testing-a-first-flow-test.md:9]:
```suggestion
Testing a flow can be done through the `fluttium test` command. This command will read the
`fluttium.yaml` file, apply any of the configuration it has to the driver, and run the given flow
file.
```

File: docs/docs/getting-started/testing-a-first-flow-test.md [hunk 12]
```diff
@@ -0,0 +1,33 @@
...
+
+Fluttium can also watch for any changes to either the flow file or the Flutter project, allowing
+us to hot reload whenever changes are detected:
+
+```shell
+fluttium test my_flow.yaml --watch
+```
+
+The `fluttium test` command has options to override settings in the `fluttium.yaml` file, for the
+full overview of options run:
```
Comment (scarletteliza) [inline docs/docs/getting-started/testing-a-first-flow-test.md:29]:
```suggestion
The `fluttium test` command has options to override settings in the `fluttium.yaml` file. For the
full overview of options run:
```

File: docs/docs/why-fluttium.md [hunk 13]
```diff
@@ -3,7 +3,7 @@ sidebar_position: 1
 description: Fluttium focuses on the real user, the tests a developer writes with Fluttium are a direct representation of the actions that an user of an application would perform.
 ---
 
-# Why Fluttium
+# 💭 Why Fluttium
 
 There are quite a few app automation frameworks out there like [Appium](https://appium.io)
 and [Maestro](https://maestro.mobile.dev), some are even dedicated to specific platforms, like
```
Comment (scarletteliza) [inline docs/docs/why-fluttium.md:9]:
```suggestion
and [Maestro](https://maestro.mobile.dev). Some are even dedicated to specific platforms, like
```

File: docs/docs/why-fluttium.md [hunk 14]
```diff
@@ -13,7 +13,7 @@ their main focus is not the [Flutter](https://flutter.dev) ecosystem.
 
 Thankfully the Flutter community has already created a beautiful set of tools to fill in the gap of
 end-to-end testing in Flutter. Tools like [Patrol](https://patrol.leancode.co/) and
-[Honey](https://github.com/clickup/honey) allow Flutter developers to easily write end-to-end tests
+[Honey](https://honey.dev/) allow Flutter developers to easily write end-to-end tests
 for their applications. But these tools have one thing in common, the tests written by a developer
 does not have to directly reflect how a real user would use an application.
```
Comment (scarletteliza) [inline docs/docs/why-fluttium.md:18]:
```suggestion
for their applications. But these tools have one thing in common: the tests written by a developer
may not directly reflect how a real user would use an application.
```

File: docs/docs/why-fluttium.md [hunk 15]
```diff
@@ -23,7 +23,7 @@ exactly where Fluttium comes into play.
 
 Fluttium focuses on the real user, the tests a developer writes with Fluttium are a direct
 representation of the actions that an user of an application would perform. Fluttium exposes a set
```
Comment (scarletteliza) [inline docs/docs/why-fluttium.md:25]:
```suggestion
Fluttium focuses on the end user. The tests a developer writes with Fluttium are a direct
representation of the actions that a user of an application would perform. Fluttium exposes a set
```

File: docs/docs/why-fluttium.md [hunk 16]
```diff
@@ -23,7 +23,7 @@ exactly where Fluttium comes into play.
...
-of actions that a developer can use to write a user flow test.
+of actions that a developer can use to write a user flow test. Developers can also define their own custom actions if needed.
 
 These user flow tests are powered by the semantic tree of the Flutter application, this allows
 Fluttium to fully act like the real user and execution actions. Fluttium optimizes these actions as
```
Comment (scarletteliza) [inline docs/docs/why-fluttium.md:29]:
```suggestion
These user flow tests are powered by the semantic tree of the Flutter application. This allows
Fluttium to fully act like the real user and execute actions. Fluttium optimizes these actions as
```

**Review by felangel — APPROVED** [review]

**Pull request merged**